src/content/development/security/reporting/_index.en.md

```diff
@@ -0,0 +1,12 @@
+---
+title: "Security Reporting"
+weight: 40
+---
+
+Submariner welcomes and appreciates responsible disclosure of security vulnerabilities.
+
+If you know of a security issue with Submariner, please report it to submariner-security@googlegroups.com.
+
+Submariner aspires to follow the [Kubernetes security reporting process](https://kubernetes.io/docs/reference/issues-security/security/),
+but is far too small of a project to implement those practices. Where applicable, Submariner will follow the principles of the Kubernetes
+process.
```

Comment:
```suggestion
Submariner aspires to follow the [Kubernetes security reporting process](https://kubernetes.io/docs/reference/issues-security/security/) where possible
but is too small of a project to fully implement those practices.
```

Reply:
This isn't part of this PR IMHO, it's copied over from the previous file. I also prefer the original version. I think it's a substantial loss to not say "Submariner will follow the principles of the Kubernetes process" - that's basically the core idea.

Reply:
If one equates "_Submariner aspires to follow_..." with "_Submariner will follow_..." then nothing is lost - I've merely included the "_where applicable_" part (as "_where possible_") in the former sentence to render the latter redundant. However if one thinks meaning is lost by losing the "_will follow_" phrasing, then we can change it to 

```suggestion
Submariner follows the [Kubernetes security reporting process](https://kubernetes.io/docs/reference/issues-security/security/) 
where possible but is too small of a project to fully implement those practices.
```

Reply:
What changes is the nuance that if you follow the link and read the K8s process, it's clearly built for a very large project. The original basically says "We're too small to follow the massive process in K8s, but we will follow the principles". The second basically says "We will try to follow the K8s process but can't implement parts of it". I think those are substantially different things.

Reply:
ok